gnucash/gnome/gnucash.appdata.xml.in.in

```diff
@@ -14,13 +14,16 @@
     </p>
     <p>With GnuCash you can (but are not limited to):</p>
     <ul>
-      <li>Keep track of your day to day personal income and expenses</li>
-      <li>Manage your stock, bond and mutual fund accounts with ease</li>
-      <li>Keep your small business' accounting up to date</li>
-      <li>Create accurate reports and graphs from your financial data</li>
-      <li>Set up scheduled transactions to avoid repeated data entry</li>
-      <li>Exchange by CSV/FinTS(former HBCI) or import SWIFT-MT9xx/QIF/OFX data including Transaction Matching</li>
-      <li>Perform financial calculations, such as a loan repayment</li>
+      <li>Keep track of your day to day personal income and expenses.</li>
+      <li>Manage your stock, bond and mutual fund accounts with ease.</li>
+      <li>Keep your small business' accounting up to date.</li>
+      <li>Create accurate reports and graphs from your financial data.</li>
+      <li>Set up scheduled transactions to avoid repeated data entry.</li>
+      <li>Process your bank data via online banking using OFX DirectConnect or FinTS.</li>
+      <li>Download Transactions and Balances and initiate various SEPA Actions using AqBanking.</li>
+      <li>Import accounts, business data, prices, and transactions from CSV and transactions and balances from many formats including CAMT, SWIFT-MT9xx, and OFX.</li>
```

Comment:
CAMT is like PAIN a subtype of UNIFI (ISO 20022)

Reply:
OK, so? It's not like most users are going to know that, and it seems to be widely used in The Netherlands. Germany too, apparently: @m53956 told me last year when I was fixing the SWIFT MT imports that he'd switched to using it because he found it more reliable.

Reply:
Users will usually read appdata.desktop before they decide to install GnuCash—a potential broad audience. S I would suggest 
… `including ISO 20022 CAMT, SWIFT-MT9xx` …
BTW, do you like the following Oxford comma?

Reply:
Yes, I'm an Oxford comma person. Do we really need the ISO spec number for CAMT?

Reply:
I think about a person reading some terms on their bank website and searching matches in accounting software.
The spec number is unique, whole googling for CAMT delivers also false results.

BTW I found some history. Since the introduction of SEPA european banks have to offer CAMT files and the requirement to offer  SWIFT MT940 will end 2025-11.

Reply:
But if few people know the spec number they won't be searching for it. Do the banks mention it on their websites or do they just say "CAMT downloads"? Searching in the US has a problem because CAMT also stands for "Corporate Annual Minimum Tax", part of the 2017 tax bill. Also it's not something that US banks do. Changing it to "camt.053 download" gets me bank pages starting on the third hit. But halfway down the first page I get a couple of banks' FAQ pages about downloading CAMT.053 data. Searching for "iso-20022 downloads" gets me pages of technical documents and nothing bank customer oriented.  Which works best in Germany? Try `"camt" downloads` to stop the search engine from finding similar words.

Reply:
Note: CAMT.05x has the same subtypes as MT-9xx.

I found also "SEPA CAMT", but  that seems eurocentric.
I started with "kontoauszug herunterladen" and got signed PDF as first choice. The signature is required by the tax office.
For "kontoauszug imortieren" I got the full range from "convert pdf" over csv, MT9xx to CAMT.
At least it seems "CAMT is sufficient here.